tools/azuredeployer/main/azure_deployer.go

```diff
@@ -149,7 +149,7 @@ func runSetupScript(ipAddress string) {
 		Timeout:         sshTimeout,
 	}
 
-	log.Print("SSH'ing into VM to complete set-up.")
+	log.Printf("SSH'ing into VM to complete set-up: %s:%s", ipAddress, sshPort)
```

Comment:
I think we should put this info on line 176, where we log `VM set-up complete.`. If the VM takes a while to spin up, this will get buried under the `Waiting for VM to be ready...` messages.